ArduCopter/Copter.h

```diff
@@ -766,6 +768,9 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // landing_gear.cpp
     void landinggear_update();
 
+    // stand_by.cpp
+    void standby_update();
```

Comment:
at the risk of being annoying, maybe we can consolidate the naming to be either "standby" or "stand_by" including the naming of the .cpp file?  I personally prefer "standby" without the underscore but I don't mind too much either way.  the justification for this is just that it will make searches easier.

Reply:
I completly agree!